🐛 [Bug report]:  Build fails due to newer version of react-loader-spinner

**Describe the bug**
A newer release of `react-loader-spinner` seems to be causing a breaking changes, thus causing the build of ReactPlay to fail.

**The build fails with the following error:**

> 14:13:31.965 | Failed to compile.
> 14:13:31.966 |  
> 14:13:31.966 | Module not found: Error: Package path ./dist/loader/css/react-spinner-loader.css is not exported from package /vercel/path0/node_modules/react-loader-spinner (see exports field in /vercel/path0/node_modules/react-loader-spinner/package.json)
> 


**Expected behavior**
Build should pass.

**Additional context**
Reverting the version of `react-loader-spinner` will fix the issue.

--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
     "react-helmet": "^6.1.0",
     "react-hot-toast": "^2.3.0",
     "react-icons": "^4.3.1",
-    "react-loader-spinner": "^5.1.5",
+    "react-loader-spinner": "~5.1.5",
     "react-organizational-chart": "^2.1.1",
     "react-qr-code": "^2.0.7",
     "react-redux": "^8.0.1",

